doc/source/tutorial/stats/sampling_tdr.rst

```diff
@@ -0,0 +1,190 @@
+
+.. _sampling-tdr:
+
+Transformed Density Rejection (TDR)
+===================================
+
+.. currentmodule:: scipy.stats
+
+* Required: T-concave PDF, dPDF
+* Optional: mode, center
+* Speed:
+
+  * Set-up: slow
+  * Sampling: fast 
+
+
+TDR is an acceptance/rejection method that uses the concavity of a transformed
+density to construct hat function and squeezes automatically. Such PDFs are
+called T-concave. Currently the following transformations are implemented:
```

Comment:
This definition is a bit steep IMO. I don't get the squeeze part. Also we should introduce slightly what is the hat function. Maybe just add an equation and show how it links to `c`

Reply:
Same as https://github.com/scipy/scipy/pull/14215#discussion_r681869291